.gitignore

```diff
@@ -8,3 +8,4 @@
 /bin/
 /composer.phar
 /nbproject
+/.idea
```

Comment:
Why did you add this?

if anything you should be removing nbproject from this file